cirq-core/cirq/ops/pauli_string.py

```diff
@@ -113,9 +113,24 @@ def __init__(
         qubit_pauli_map: Optional[Dict[TKey, 'cirq.Pauli']] = None,
         coefficient: 'cirq.TParamValComplex' = 1,
     ):
-        """Initializes a new PauliString.
+        """Initializes a new `PauliString` operation.
+
+        `cirq.PauliString` represents a multi-qubit pauli operator, i.e.
```

Comment:
I think this should be at the class level docstring?

Reply:
Make it class level